pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -199,6 +201,9 @@ func (o *OvirtMapper) mapNics() []kubevirtv1.Interface {
 		if nicInterface, ok := nic.Interface(); ok {
 			kubevirtNic.Model = string(nicInterface)
 		}
+		// FIXME: should this be as part of mapping?
+		// options: slirp, bridge, sriov and masquerade
```

Comment:
VM migration works only for `masquerade`. @phoracek any suggestions?

Reply:
Any context on what's going on here?

Reply:
This code imports vms from ovirt to kubevirt. The question is about which option to use.

Reply:
If you want to be always able to import oVirt VM to KubeVirt, you should just give it a single network interface that would be connected through `masquerade` to the pod network.

If you really want to give the VM the same environment as it had on oVirt, somebody needs to configure the L2 network (possibly via kubernetes-nmstate with bridges representing each oVirt network) and configure NetworkAttachmentDefiniton per each. Then you would add one network interface with `bridge` binding connected to respective multus network.

Reply:
We do not want to configure anything. We expect it to be preconfigured, if it is not there we should fail the process. As I understand we should use `masquerade` for pod network and `bridge` for multus networks. Please correct me if I am wrong.

Reply:
That's right. So what you can do is to add Interface to the VM which would be based on a Multus network and would use bridge binding.